### PR TITLE
[NOTEPAD] Arabic also uses Right-to-Left (RTL) layout

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
The Arabic user uses Right-to-Left (RTL) layout. Notepad should support RTL.
JIRA issue: [CORE-18837](https://jira.reactos.org/browse/CORE-18837)

## Proposed changes

- Use `PRIMARYLANGID` macro to get the primary language ID.
- Add Arabic case.

### CODE REVIEW

File: base/applications/notepad/main.c [hunk 1]
```diff
@@ -573,14 +573,13 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE prev, LPTSTR cmdline, int sh
...
-    case MAKELANGID(LANG_HEBREW, SUBLANG_DEFAULT):
-        SetProcessDefaultLayout(LAYOUT_RTL);
-        break;
-
-    default:
-        break;
+        case LANG_ARABIC:
+        case LANG_HEBREW:
+            SetProcessDefaultLayout(LAYOUT_RTL);
+            break;
```
Comment (learn-more) [inline base/applications/notepad/main.c:582]: Why is this required, this seems like something that should be done by the OS?
Does it need this when running our Notepad on an RTL Windows machine?

Comment (HBelusca) [reply base/applications/notepad/main.c:582]: It's automatically done by the OS **only if** the file description string version resource contains two left-to-right marks, as done e.g. in https://git.reactos.org/?p=reactos.git;a=blob;f=base/setup/welcome/lang/he-IL.rc;hb=5bc045c041f0c90d54f009c622d085422c80b662
See also https://devblogs.microsoft.com/oldnewthing/20100611-00/?p=13743

Comment (katahiromz) [reply base/applications/notepad/main.c:582]: https://github.com/reactos/reactos/blob/2f43d9c18338722999f18630dabd77821a1d79cb/base/applications/notepad/rsrc.rc#L26

Comment (katahiromz) [reply base/applications/notepad/main.c:582]: https://github.com/reactos/reactos/blob/35a816a24e3b7b5752a287686c9df833af6d07a9/sdk/include/reactos/version.rc#L121

**Comment by HBelusca** [comment]
Actually there is no point in having this PR, because literally **no** ReactOS application is translated in arabic so far, except for OSK.

**Comment by katahiromz** [comment]
Superceded by #5138. Closed.
